torchgeo/datasets/agb_live_woody_density.py

```diff
@@ -115,6 +115,10 @@ def _verify(self) -> None:
 
     def _download(self) -> None:
         """Download the dataset."""
+
+        if(isinstance(self.paths,os.PathLike)):
+            self.paths = str(self.paths)
```

Comment:
Is it possible to make them work if we keep them as pathlib instead of casting them?

Reply:
Unfortunately, it isn't possible to keep it as pathlib because of the method calling `` download_url(self.url, self.paths, self.base_filename)`` which is a torchvision method and only accepts strings.